api/lib/Utils.js

```diff
@@ -118,6 +118,19 @@ let Utils = {
       }
       return acc;
     }, {});
+  },
+
+  compareDateStrings(record1, record2) {
+    const dates = [record1, record2].map(record => {
```

Comment:
I think we can make it even simpler by saying date1 = new Date(record1), date2 = new Date(record2)